react/Gallery.js

```diff
@@ -38,7 +39,7 @@ const Gallery = ({
     return maxItemsPerRow <= maxItems ? maxItemsPerRow : maxItems
   }
 
-  const itemsPerRow =
+  const itemsPerRow = !useDefaultDistribution ? maxItemsPerRow :
```

Comment:
There's a side-effect to this implementation.
I don't know if you want this behavior, but if a user were to set this `useDefaultDistribution` prop to `false`, the `LayoutSwitcher` component would be useless, as this conditional check would not try to validate `layoutMode === 'small'`.

Reply:
Also, I think this bit of code looks confusing with 3 ternary conditions in a row, a refactor would be welcome.

Reply:
Check this change.

``` javascript
  const defaultDistribution = layoutMode === 'small'
  ? TWO_COLUMN_ITEMS
  : isMobile
  ? ONE_COLUMN_ITEMS
  : getItemsPerRow() || maxItemsPerRow

  const itemsPerRow = !useDefaultDistribution ? maxItemsPerRow : defaultDistribution
```

Reply:
Nice! That solves the layout on mobile :)
Would you guys please just run `yarn lint --fix`?

Reply:
Sorry, We run `yarn  lint --fix`  in the project but get this error

 ```
yarn run v1.19.1
error Command "lint" not found.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
```

The search-result project only have this scrips

```
"scripts": {
    "pretest": "yarn",
    "test": "vtex-test-tools test"
  },
```

We run the `test` script and no have errors with the change. 
Is what was done right or wrong? or what should I do